repour/adjust/gradle_provider.py

```diff
@@ -39,6 +39,10 @@ def adjust(work_dir, extra_adjust_parameters, adjust_result):
             temp_build_parameters.append(
                 "-DrestRepositoryGroup=" + specific_indy_group)
 
+        extra_parameters, subfolder = yield from util.get_extra_parameters(extra_adjust_parameters)
+
+        repo_dir = os.path.join(repo_dir, subfolder)
```

Comment:
sorry I am 0 on python, I am missing why LN#44 is needed? repo_dir seems not used?

Reply:
Well, this is a good catch. It's a copy/paste error, in PME it is `repo_dir`, in GME it's `work_dir`. Fixing a second.

Reply:
This should be fixed now.